dpp/src/schema/data_contract/dataContractMeta.json

```diff
@@ -445,4 +482,4 @@
     "documents"
   ],
   "additionalProperties": false
-}
+}
```

Comment:
Please don't forget about the newlines! 🙂

Reply:
fixed